packages/wallets/src/components/DerivAppsSection/DerivAppsTradingAccount.tsx

```diff
@@ -7,22 +7,109 @@ import { LabelPairedArrowUpArrowDownSmBoldIcon } from '@deriv/quill-icons';
 import { WalletDisabledAccountModal } from '../WalletDisabledAccountModal';
 import { WalletListCardBadge } from '../WalletListCardBadge';
 import { WalletMarketIcon } from '../WalletMarketIcon';
 import { WalletStatusBadge } from '../WalletStatusBadge';
 
+type TDerivAppsTradingAccountButtonContent = {
+    activeTradingAccount: THooks.TActiveLinkedToTradingAccount;
+    isEuRegion: boolean;
+    mfAccountStatusDetails?: ReturnType<typeof useWalletsMFAccountStatus>['data'];
+};
+
+const DerivAppsTradingAccountButtonContent: React.FC<TDerivAppsTradingAccountButtonContent> = ({
+    activeTradingAccount,
+    isEuRegion,
+    mfAccountStatusDetails,
+}) => {
+    const history = useHistory();
+    const { show } = useModal();
+    const { is_disabled: isAccountDisabled, is_virtual: isDemo, loginid } = activeTradingAccount ?? {};
+    const {
+        client_kyc_status: clientKycStatus,
+        is_added: isMFAccountAdded,
+        mfAccountStatus,
+    } = mfAccountStatusDetails ?? {};
+
+    if (isAccountDisabled) {
+        return <WalletStatusBadge badgeSize='md' padding='tight' status='disabled' />;
+    }
+
+    if (isEuRegion && !isDemo && mfAccountStatus && isMFAccountAdded && clientKycStatus) {
+        return (
+            <ClientVerificationStatusBadge
+                onClick={() =>
+                    show(
+                        <ClientVerificationModal
+                            account={{
+                                client_kyc_status: clientKycStatus,
+                                is_added: isMFAccountAdded,
+                            }}
+                        />
+                    )
+                }
+                variant={mfAccountStatus}
+            />
+        );
+    }
+
+    return (
+        <button
+            className='wallets-deriv-apps-section__button'
+            data-testid='dt_deriv-apps-trading-account-transfer-button'
+            onClick={() => {
+                history.push('/wallet/account-transfer', {
+                    toAccountLoginId: loginid,
+                });
+            }}
+        >
+            <LabelPairedArrowUpArrowDownSmBoldIcon />
+        </button>
+    );
+};
+
+const AccountBalance: React.FC<{ activeTradingAccount: THooks.TActiveLinkedToTradingAccount }> = ({
+    activeTradingAccount,
+}) => {
+    const { data: balanceData, isLoading: isBalanceLoading } = useAllBalanceSubscription();
+    const balance = balanceData?.[activeTradingAccount?.loginid ?? '']?.balance;
+
+    return isBalanceLoading ? (
+        <div
+            className='wallets-skeleton wallets-deriv-apps-balance-loader'
+            data-testid='dt_deriv-apps-balance-loader'
+        />
+    ) : (
+        <Text align='start' size='sm' weight='bold'>
+            {displayMoney(balance, activeTradingAccount?.currency_config?.display_code, {
+                fractional_digits: activeTradingAccount?.currency_config?.fractional_digits,
+            })}
```

Comment:
@jim-deriv I've created WalletMoney component for this purposes, could you try to use it?😅 It's also supports RTL.